cmd/update-main.go

```diff
@@ -68,90 +68,110 @@ const (
 	minioReleaseURL           = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
 )
 
-func getCurrentReleaseTime(minioVersion, minioBinaryPath string) (releaseTime time.Time, err error) {
-	if releaseTime, err = time.Parse(time.RFC3339, minioVersion); err == nil {
+type updateParams struct {
+	versionStr         string
+	serverBinaryPath   string
+	cgroupFile         string
+	releaseChecksumURL string
+	serverMode         string
+}
```

Comment:
`updateParams` does not look like the right name for this.
i.e For example why is cgroupFile part of updateParams
why is isDockerInternal a method on updateParams

If the structure name and method names are changed so that they look related then it will look better